### PR TITLE
Feature/347 UI fixes v2

### LINKED ISSUE #347: Admin Portal - Subscriptions Horizontal Scroll missing
**Describe the bug**
Subscriptions list does not have horizontal scroll bar on lower resolution / high DPI settings.
This can cause confusion and block user action since the action panel for subscription is not visible

**To Reproduce**
1. Go to Subscriptions page
2. Resize the window to a smaller size

**Expected behavior**
Able to scroll the table and see entire row

**Screenshots**
image

### PR DESCRIPTION
**Pull request by alexdrenea**
**Admin Portal**
- Fixed scroll issue on homepage due to fixes header
- SubscriptionsPage: Horizontal scroll on subscriptions table
image

- SubscriptionLogDetails: Vertical scroll on logs popup
- LogDetails, ChangePlan, ChangeQuantity modals rendered correctly (using PartialView not Full View)
image

**Customer Portal**
- SubscriptionLogDetails: Vertical scroll on logs popup
image

- SubscriptionLogDetails: Fix issue when no logs present
image

- LogDetails, ChangePlan, ChangeQuantity modals rendered correctly (using PartialView not Full View)

fixes #347

### CODE REVIEW

File: src/CustomerSite/Views/Home/SubscriptionLogDetail.cshtml [hunk 1]
```diff
@@ -10,9 +10,9 @@
             <div class="card card-header mb0 pb0">
                 <dl class="row">
                     <dt class="col-2 text-right"><b>Plan</b></dt>
-                    <dd class="col-3 text-left">@Model.First().Subscription.AmpplanId</dd>
+                    <dd class="col-3 text-left">@Model.FirstOrDefault()?.Subscription?.AmpplanId</dd>
```
[the remark this answers is not in the document]
Comment (alexdrenea) [reply src/CustomerSite/Views/Home/SubscriptionLogDetail.cshtml:13]: @santhoshb-msft  it is fixed in the AdminPortal, this is just applying the fix for the Customer portal

**Comment by santhoshb-msft** [comment]
@alexdrenea - could you please create an issue item and attach to this PR please.

**Comment by neelavarshad** [comment]
Verification Checks: 
Horizontal scrollbar
image

Log Details 
image

CustomerSite
 
image

image

**Comment by neelavarshad** [comment]
@santhoshb-msft to merge these changes

**Comment by santhoshb-msft** [comment]
@alexdrenea could you please attach an issue to this PR

**Comment by alexdrenea** [comment]
Fixes #347

**Comment by santhoshb-msft** [comment]
Thank you for the approval @neelavarshad, merging this as advised.

**Pull request merged**